### DISCUSSION TITLE
How can I change  SSH user ?

### DISCUSSION THREAD

**Discussion by IgordeOliveira** [serversideup/spin#56] [Q&A] [answered]
Hi, I cannot connect to my VM with the error "Failed to connect to the host via ssh: ssh: Could not resolve hostname ...", wich I think could be the user for SSH, this VM uses ubuntu as user, not root as usual. Its possible change ?

**Comment by jaydrogers** [comment, marked as answer]
Thanks for bringing this to my attention. This is something that I forgot to document.

There is a `--user` flag that can be set https://github.com/serversideup/spin/blob/200bc7d48549754ea7463f2ad829fb9df0e970cb/lib/actions/provision.sh#L16

For example:

```
spin provision --user ubuntu
```

That should set the remote SSH user as `ubuntu` for you.

I will get this added to the docs.
